src/app/app.component.html

```diff
@@ -11,6 +11,6 @@
   <app-courses></app-courses>
 </section>
 
```

Comment:
app.component for whole application. Maybe be better to move breadcrumbs and app-search to 'course' component? Because 'login' page won't be include these components.